.editorconfig

```
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
```

Let's use `4` spaces.

```suggestion
indent_size = 4
```